### ISSUE TITLE
Using API to get team list of user returns empty array

### ISSUE THREAD

**Issue by dataviruset** [grafana/grafana#14355]
### What Grafana version are you using?
grafana/grafana:5.4.0

### What datasource are you using?
InfluxDB, ElasticSearch

### What OS are you running grafana on?
CentOS 7.5 in Docker, kernel 3.10.0-514.el7.x86_64

### What did you do?
Queried /api/users/9/teams with a local account to get the team the user ID 9 is a member of. User ID 9 is from LDAP.

### What was the expected result?
An array with a team I had assigned to the user with user ID 9.

### What happened instead?
An empty array

```
$ curl -H "Authorization: Basic ..." -s "http://grafana.example.com/api/users/9/teams" | python -m json.tool
[]

$ curl -H "Authorization: Basic ..." -s "http://grafana.example.com/api/users/9" | python -m json.tool
{
"email": "blah@example.com",
"id": 9,
"isGrafanaAdmin": false,
"login": "blah",
"name": "\u8bd7\u5f3a \u9ec4",
"orgId": 1,
"theme": ""
}

$ curl -H "Authorization: Basic ..." -s "http://grafana.example.com/api/teams/2/members" | python -m json.tool
[
...
{
"avatarUrl": "/avatar/blah",
"email": "blah@example.com",
"labels": [],
"login": "blah",
"orgId": 1,
"teamId": 2,
"userId": 9
},
]
```

My account used for querying:

```
$ curl -H "Authorization: Basic ..." -s "http://grafana.example.com/api/user" | python -m json.tool
{
"email": "me@example.com",
"id": 122,
"isGrafanaAdmin": true,
"login": "me",
"name": "me",
"orgId": 1,
"theme": ""
}
```

**Comment by torkelo** [comment]
![image](https://user-images.githubusercontent.com/10999/50589949-faf1c300-0e88-11e9-9031-dbb848fbd64a.png)

Strange unable to replicate.

**Comment by MaksimPuzyrkov** [comment]
Same issue here. Can get user organisations, but teams returns an empty array.

Grafana v5.4.2 (commit: d812109)
Docker version 18.09.1, build 4c52b90
CentOS Linux release 7.5.1804 (Core), 3.10.0-862.11.6.el7.x86_64

Authentication via generic_oauth using AzureAD
2 Grafana nodes in a cluster behind reverse proxy with MariaDB shared database

```
 ➤ curl -k -u admin:secret https://grafana.example.net/api/teams/21
{"id":21,"orgId":1,"name":"Test Team","email":"","avatarUrl":"/avatar/9f6131de81a0445c6897b1e82436ea93","memberCount":1}
➤ curl -k -u admin:secret https://grafana.example.net/api/teams/21/members
[{"orgId":1,"teamId":21,"userId":54,"email":"USER@example.com","login":"USER@example.com","avatarUrl":"/avatar/59932340bc1dfaeb24f6046cfa71e596","labels":[]}]
➤ curl -k -u admin:secret https://grafana.example.net/api/users/54/
{"id":54,"email":"USER@example.com","name":"User Example","login":"USER@example.com","theme":"","orgId":1,"isGrafanaAdmin":false}
➤ curl -k -u admin:secret https://grafana.example.net/api/users/54/orgs
[{"orgId":1,"name":"Example Organisation","role":"Viewer"}]
➤ curl -k -u admin:secret https://grafana.example.net/api/users/54/teams
[]
```

**Comment by marefr** [comment]
Probably fixed by #15593
